src/components/ChattingBubble/index.tsx

```diff
@@ -0,0 +1,62 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface ChattingBubbleProps extends HTMLAttributes<HTMLDivElement> {
+  user: 'sender' | 'receiver'
+  children: string
+}
+
+interface StyledBubbleProps {
+  user: 'sender' | 'receiver'
+}
+
+export const ChattingBubble = ({
+  user,
+  children,
+  ...props
+}: ChattingBubbleProps): ReactElement => {
+  const redex = /\s/gi
```

Comment:
```suggestion
  const newLineRegex = /\s/gi
```
개행에 대한 regex라면 위와 같이 변경하는 게 좋을 것 같숨니다!